.github/workflows/test.yml

```
@@ -0,0 +1,14 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+        with:
+          haxe-version: 4.0.5
```

I think this is a remnant of a previous iteration.

```suggestion
        with:
          haxe-version: 4.0.5
```

---

Whoops, absolutely right.